### PR TITLE
Start of updating the code of conduct

### PR DESCRIPTION
**Pull request by bedroesb**

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -106,24 +102,19 @@ If a participant engages in behaviour that violates this Code of Conduct, any me
 ### 2.4 Feedback
 
 This Code of Conduct is not intended as a static set of rules by which everyone must abide.
-Rather, you are invited to make suggestions for updates or clarifications by contacting [the CoC Oversight Committee](mailto:coc.oversight@elixir-europe.org) or by making a pull request to this document on GitHub.
+Rather, you are invited to make suggestions for updates or clarifications by contacting [RDM CoC Group](mailto:rdm-coc@elixir-europe.org) or by making a pull request to this document on GitHub.
```
Comment (nsoranzo) [inline CODE_OF_CONDUCT.md:105]:
```suggestion
Rather, you are invited to make suggestions for updates or clarifications by contacting the [RDM CoC Group](mailto:rdm-coc@elixir-europe.org) or by making a pull request to this document on GitHub.
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -133,18 +124,13 @@ This is the enforcement manual followed by the ELIXIR RDM Toolkit project team.
 It's used when we respond to an issue to make sure we're consistent and fair.
 Enforcement of the Code of Conduct should be respectful and not include any harassing behaviours.
 
-### 4.1 The Code of Conduct Oversight Committee
+### 4.1 The RDM Code of Conduct Group
 
-The ELIXIR Code of Conduct Oversight committee is:
+The RDM Code of Conduct group is:
```
Comment (nsoranzo) [inline CODE_OF_CONDUCT.md:129]:
```suggestion
The members of the RDM Code of Conduct group are:
```

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -169,18 +155,18 @@ Upon receiving a report of an incident, the ELIXIR Code of Conduct Oversight Com
...
+The RDM CoC group will aim to have a resolution agreed upon within one week.
+In the event that a resolution can't be determined in that time, a member of the RDM CoC group will respond to the reporter(s) with an update and projected timeline for resolution.
 
 ### 4.4 Resolutions
 
-The ELIXIR Code of Conduct Oversight Committee  will seek to agree on a resolution by consensus of all members investigating the report in question.
+The RDM CoC group will seek to agree on a resolution by consensus of all members investigating the report in question.
 If the committee cannot reach consensus and deadlocks for over a week, Niklas Blomberg, as Director of ELIXIR, will break the tie.
-If Dr Blomberg is unable to take part in the discussion due to a conflict of interest, Susanna Rep, as an external member of the ELIXIR Code of Conduct Oversight committee, will make the decision.
+If Dr Blomberg is unable to take part in the discussion due to a conflict of interest, Susanna Rep, as an external member of the RDM CoC group, will make the decision.
```
Comment (nsoranzo) [inline CODE_OF_CONDUCT.md:169]:
```suggestion
If Dr Blomberg is unable to take part in the discussion due to a conflict of interest, Susanna Repo, as an external member of the RDM CoC group, will make the decision.
```

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -169,18 +155,18 @@ Upon receiving a report of an incident, the ELIXIR Code of Conduct Oversight Com
...
+The RDM CoC group will aim to have a resolution agreed upon within one week.
+In the event that a resolution can't be determined in that time, a member of the RDM CoC group will respond to the reporter(s) with an update and projected timeline for resolution.
 
 ### 4.4 Resolutions
 
-The ELIXIR Code of Conduct Oversight Committee  will seek to agree on a resolution by consensus of all members investigating the report in question.
-If the committee cannot reach consensus and deadlocks for over a week, Niklas Blomberg, as Director of ELIXIR, will break the tie.
-If Dr Blomberg is unable to take part in the discussion due to a conflict of interest, Susanna Rep, as an external member of the ELIXIR Code of Conduct Oversight committee, will make the decision.
+The RDM CoC group will seek to agree on a resolution by consensus of all members investigating the report in question.
+If the committee cannot reach consensus in over a week, Niklas Blomberg, as Director of ELIXIR, will descide on an appropriate resolution.
```
Comment (nsoranzo) [inline CODE_OF_CONDUCT.md:168]:
```suggestion
If the committee cannot reach consensus within a week, Niklas Blomberg, as Director of ELIXIR, will decide on an appropriate resolution.
```

Comment (nsoranzo) [reply CODE_OF_CONDUCT.md:168]: This comment is still standing.

Comment (bedroesb) [reply CODE_OF_CONDUCT.md:168]: @nsoranzo I was a bit to quick merging this. I updated it in the master branch. Thank you for checking out on this!

**Pull request merged**